src/main/java/de/hysky/skyblocker/skyblock/dungeon/LividColor.java

```diff
@@ -39,4 +59,32 @@ public static void update() {
             }
         }
     }
+
+    private static void onLividColorFound(Block color) {
+        LividColor.color = WOOL_TO_FORMATTING.get(color);
+        String colorString = Registries.BLOCK.getId(color).getPath();
+        MessageScheduler.INSTANCE.sendMessageAfterCooldown(SkyblockerConfigManager.get().locations.dungeons.lividColor.lividColorText.replace("[color]", colorString.substring(0, colorString.length() - 5)));
+        tenTicks = 0;
+    }
+
+    public static boolean shouldGlow(Entity armorStand) {
```

Comment:
Should be refactored to be similar to `getGlowColor` instead of checking formatting.